packages/manager/apps/web/client/app/hosting/multisite/git-association/hosting-multisite-git-association.html

```diff
@@ -0,0 +1,136 @@
+        </oui-field>
+    </div>
+    <h3 data-translate="hosting_multisite_git_association_configure"></h3>
+
+    <oui-message type="info">
+        <p
+            class="font-weight-bold"
+            data-translate="hosting_multisite_git_association_configure_info_title"
+        ></p>
+        <ul>
+            <li
+                data-translate="hosting_multisite_git_association_configure_info_description"
+            ></li>
+            <li
+                data-translate="hosting_multisite_git_association_configure_info_https"
+            ></li>
+            <li
+                data-translate="hosting_multisite_git_association_configure_info_ssh"
+            ></li>
+        </ul>
+        <span
+            class="bold"
+            data-translate="hosting_multisite_git_association_configure_info_empty_repo"
+        ></span>
+        <span
+            data-translate="hosting_multisite_git_association_configure_info_not_empty_repo"
+        ></span>
+        <br />
+        <br />
+        <p
+            class="font-weight-bold"
+            data-translate="hosting_multisite_git_association_configure_info_add_in_paremeters"
+        ></p>
+    </oui-message>
+
+    <div class="row">
+        <oui-field
+            class="col-md-4"
+            data-label="{{ ::'hosting_multisite_git_association_repository' | translate }}"
+        >
+            <div class="d-flex oui-input-group oui-input-group_inline">
+                <input
+                    type="text"
+                    id="repository_url"
+                    name="repository_url"
+                    class="oui-input mr-2"
+                    data-ng-pattern="/^[https:\/\/|git:\/\/].*/"
+                    data-ng-model="$ctrl.model.repositoryUrl"
+                    data-ng-class="{'oui-field_error': $ctrl.gitAssociationForm.repository_url.$invalid}"
+                    placeholder="{{ $ctrl.repositoryPlaceholder }}"
+                    required
+                />
+            </div>
+            <div class="d-flex">
+                <small
+                    class="w-100"
+                    data-ng-bind-html="'hosting_multisite_git_association_ssh_url_repo_help' | translate : { httpsRepo: $ctrl.EXAMPLE_HTTPS_REPOSITORY_URL, sshRepo: $ctrl.EXAMPLE_SSH_REPOSITORY_URL }"
+                ></small>
+            </div>
+        </oui-field>
+    </div>
+
+    <div class="row">
+        <oui-field
+            class="col-md-4"
+            data-label="{{ ::'hosting_multisite_git_association_branch' | translate }}"
+        >
+            <div class="d-flex oui-input-group oui-input-group_inline">
+                <input
+                    type="text"
+                    id="branch"
+                    name="branch"
+                    required
+                    class="oui-input mr-2"
+                    data-ng-model="$ctrl.model.branchName"
+                    data-ng-class="{'oui-field_error': $ctrl.gitAssociationForm.branch.$invalid}"
```

Comment:
I think you can use data-ui-validate i think the class is directly manage by oui-field